src/main/java/duke/InputHandler.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.TaskStorage;
+
+class InputHandler {
```

Comment:
Perhaps add a header comment for the class InputHandler as it seems to be a requirement.